### PR TITLE
hw2

### PR DESCRIPTION
**Pull request by mikhnovich**

### CODE REVIEW

File: src/test/java/hw2/FirstExercise.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        List<String> actualLeftSideMenuNames = getMenuItemsText(leftSideMenuItems);
+        List<String> expectedLeftSideMenuNames = Arrays.asList(
+                "Home",
+                "Contact form",
+                "Service",
+                "Metals & Colors",
+                "Elements packs"
+        );
+        asrt.assertEquals(actualLeftSideMenuNames, expectedLeftSideMenuNames);
+
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw2/FirstExercise.java:92]: How we will know that our assertions passed?

File: src/test/java/hw2/SecondExercise.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        new Select(colorsDropdown).selectByVisibleText("Yellow");
+
+        //9. Log rows are displayed and:
+        // checkbox name and its status are corresponding to selected
+        // radio button name and it status is corresponding to selected
+        // dropdown name and selected value is corresponding to selected
+
+        WebElement windLog = driver.findElement(By.xpath("//*[contains(text(),'Wind: condition changed to true')]"));
+        WebElement waterLog = driver.findElement(By.xpath("//*[contains(text(),'Water: condition changed to true')]"));
+        WebElement yellowColorLog = driver.findElement(By.xpath("//*[contains(text(),'Colors: value changed to Yellow')]"));
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw2/SecondExercise.java:48]: Why do you decide use full log text in the locator? Why do you do not check the text of the locator?

File: src/test/java/hw2/FirstExercise.java (hunk 1 above)
Comment (mikhnovich) [reply src/test/java/hw2/FirstExercise.java:93]: We can add assertAll() to the end of the test, than the test will fail if one of the assertions inside fails.

File: src/test/java/hw2/SecondExercise.java (hunk 2 above)
Comment (mikhnovich) [reply src/test/java/hw2/SecondExercise.java]: corrected locator:
- the search is inside the "log list panel" elements 
- the locator is looking for keywords in phrases inside the log list panel

File: src/test/java/hw2/SecondExercise.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        // radio button name and it status is corresponding to selected
+        // dropdown name and selected value is corresponding to selected
+
+        WebElement windLog = driver.findElement(By.xpath("//li[contains(text(),'Water') and contains(text(),'true')]"));
+        WebElement waterLog = driver.findElement(By.xpath("//li[contains(text(),'Water') and contains(text(),'true')]"));
+        WebElement yellowColorLog = driver.findElement(By.xpath("//li[contains(text(),'Colors') and contains(text(),'Yellow')]"));
+
+        assertTrue(windLog.isDisplayed());
+        assertTrue(waterLog.isDisplayed());
+        assertTrue(yellowColorLog.isDisplayed());
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw2/SecondExercise.java:52]: Why do you do not check text of the log?

```
Assert that 
•	for each checkbox there is an individual log row and value is corresponded to the status of checkbox
•	for radio button there is a log row and value is corresponded to the status of radio button
•	for dropdown there is a log row and value is corresponded to the selected value. 
```

Comment (mikhnovich) [reply src/test/java/hw2/SecondExercise.java]: Done

**Pull request merged**